### DISCUSSION TITLE
GUI: DIANN exits before search start

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#389] [Q&A] [answered]
On a new computer, I just installed the most recent version of DIANN. 
However, the searches don't start as DIANN exits the search within seconds. There is no error. No file is written, including no log file. 

I loaded a pipeline file that worked on another computer ( with identical raw files and libraries on a network drive accessible to both computers), however still the same happened
![image](https://user-images.githubusercontent.com/98888664/169539680-4fad31af-bba9-4912-ab3d-7656da73a4fa.png)

**Comment by vdemichev** [comment]
It's DIA-NN 1.8, not 1.8.1. Please try 1.8.1. Please run diann.exe from the command line and see what the output is. Please also try to create a minimal reproducible example of the problem.

**Reply by JM-Bader** [reply]
The pipeline file worked with DIANN 1.8 on another computer, so it should also work on this one. 
Anyway, I tried to install DIANN1.8.1. However, I get this error message immediately when executing the setup exe:
![image](https://user-images.githubusercontent.com/98888664/169651246-423558f3-433e-4749-a712-541f895f03be.png)

**Comment by vdemichev** [comment, marked as answer]
Please make sure the setup file is on a local hard drive, e,g. in C:/Temp/

**Reply by JM-Bader** [reply]
1) this resolved the installation issues.
2) With DIANN 1.8.1 the search actually starts without issues.
